Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,14 +1,46 @@
-mat-card-content {
-  margin: 0 16px;
+@import 'mixins';
+@import 'media-queries';
+@import '~@angular/material/theming';
 
-  h2 {
-    margin: 16px 0 0 0;
-  }
+.header {
+  text-align: center;
+}
+
+mat-card {
+  text-align: center;
+  margin: 0.5%;
+  border: 1.5px solid;
 
-  #contribute {
-    display: flex;
-    span {
-      margin-left: 10px;
+  mat-card-header {
+    //place of the icon
+    justify-content: center;
+    mat-icon {
+      @include mat-icon-size(60px);
     }
   }
+
+  mat-card-title {
+    line-height: 1;
+  }
+}
+
+.small-card {
+  width: 50%;
+}
+
+//style for the contributors table
```

Comment:
How about grouping them into one class? So that it will be obvious that all of these sytle the contributers table?